.github/workflows/docs_ci.yml

```diff
@@ -16,3 +16,31 @@ jobs:
         run: ./v check-md -hide-warnings .
       ## NB: -hide-warnings is used here, so that the output is less noisy,
       ## thus real errors are easier to spot.
+
+  report-missing-fn-doc:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build V
+        run: make
+
+      - name: Checkout previous v
+        uses: actions/checkout@v2
+        with:
+          repository: vlang/v
+          path: pv
+
+      - name: Check against parent commit
+        run: |
+          ./v run cmd/tools/missdoc.v ./vlib/ > /tmp/n_v.txt
+          ./v run cmd/tools/missdoc.v ./pv/vlib/ > /tmp/o_v.txt
+          count_new=$(cat /tmp/n_v.txt | wc -l)
+          count_old=$(cat /tmp/o_v.txt | wc -l)
+          if [[ ${count_new} -gt ${count_old} ]]; then
```

Comment:
does that test numerically or alphabetically?

Reply:
although for numbers that may not matter actually

Reply:
Should be numerically - but bash and `[`/`[[`/`(`/`((` is *tricky* AF

Reply:
https://stackoverflow.com/a/42745537/1904615 <- should be numeric `-gt` vs. `<`